src/core/errors/MessagePlaceholder.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <Toast />
+</template>
+
+<script lang="ts">
+import { defineComponent, watch } from 'vue';
+import store from '@/store';
+import { StoreModuleEnum } from '@/store/types';
+import Toast from 'primevue/toast';
+import { useToast } from 'primevue/usetoast';
+import { MessagesGetterEnum, MessagesI } from '@/core/messages/store/types';
+
+export default defineComponent({
+  name: 'MessagePlaceholder',
+  components: {
+    Toast,
+  },
+  setup() {
+    const toast = useToast();
+    const showError = (message: string, severity: string, status: string): void => {
+      toast.add({
+        severity: severity,
+        summary: message,
+        detail: status,
+        life: 6000,
+      });
+    };
+
+    watch(
+      () => store.getters[`${StoreModuleEnum.messagesStore}/${MessagesGetterEnum.getMessages}`],
+      function () {
```

Comment:
It would be better no to watch getter results, but to create computed property that will change when store is changed